app/components/forms/PlannerForm.tsx

```diff
@@ -4,10 +4,11 @@ import React, { useState } from 'react';
 import { useTaskStore } from '@/lib/store/task';
 import TagSelector from '../AddTasks/TagSelector';
 
-export interface PlannerTask {
+interface PlannerTask {
   text: string;
   date?: string;
   tag?: string;
+  priority?: number; // Int in the table - using as a boolean for now, but we might want levels to priority so an Int made more sense
 }
 
 type SectionKeys = 'contact' | 'schedule' | 'followUp' | 'research' | 'create';
```

Comment:
I think component level types (that aren't exported and passed around) are fine. As soon as these need to be exported we should look at centralizing them better, though.